bpf/init.sh

```diff
@@ -34,6 +34,9 @@ set -x
 # Enable JIT
 echo 1 > /proc/sys/net/core/bpf_jit_enable
 
+# Disable rp_filter
+echo 0 > /proc/sys/net/ipv4/conf/all/rp_filter
```

Comment:
No, it turns out if `all` is set to `1` it doesn't matter what's the state of the remaining interfaces.